go/tools/builders/buildid.go

```diff
@@ -0,0 +1,20 @@
+package main
```

Comment:
Need a copyright notice and some top-level documentation for this tool.

Reply:
got it